docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
   prometheus-pushgateway:
     image: prom/pushgateway
     ports:
-      - 19090:9090
+      - 19091:9091
```

Comment:
This will be filtered by firewall on GCE, but otherwise it would be exposed. Better use `127.0.0.1:19091:9091`.

Reply:
`19091` port is only used on that host, e.g. to get a list of available metrics.  It shouldn't be exposed publicly.

Reply:
But it is. Unless you explicitly specify host IP, listening on `0.0.0.0` aka _all interfaces_ is assumed. Again, this is currently filtered by default firewall rules on GCE. Also, this was there before you commit.

Reply:
By publicly, I mean outside of Google Cloud.

Reply:
It'll be exposed outside of Google Cloud if firewall rules on GCE won't filter it out. I'm not sure what's reachable via this port but let's just say this relies on properly configured firewall.

Reply:
Changed.